src/06_tuples.py

```diff
@@ -35,10 +35,12 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(tuple):
+    print(tuple)
```

Comment:
Actually looking for a for loop here. Printing out each item in the tuple, not the whole tuple in one go.